homeassistant/config.py

```diff
@@ -37,6 +42,31 @@
 }
 
 
+def _valid_customize(value):
+    """Config validator for customize."""
+    if not isinstance(value, dict):
+        raise vol.Invalid('Expected dictionary')
+
+    for key, val in value.items():
+        if not valid_entity_id(key):
+            raise vol.Invalid('Invalid entity ID: {}'.format(key))
+
+        if not isinstance(val, dict):
+            raise vol.Invalid('Value of {} is not a dictionary'.format(key))
+
+    return value
+
+CORE_CONFIG_SCHEMA = vol.Schema({
+    CONF_NAME: vol.Coerce(str),
+    CONF_LATITUDE: cv.latitude,
+    CONF_LONGITUDE: cv.longitude,
+    CONF_TEMPERATURE_UNIT: cv.temperature_unit,
+    CONF_TIME_ZONE: cv.time_zone,
+    vol.Required(CONF_CUSTOMIZE,
```

Comment:
Does this state that we must have a customize section in the core home-assistant config?